versioned_sidebars/version-2.0.0-sidebars.json

```diff
@@ -59,10 +57,10 @@
           "collapsible": true,
           "collapsed": true,
           "items": [
+            "quickstart/samples-mux",
             "quickstart/samples-echo",
             "quickstart/samples-gin",
             "quickstart/samples-redis",
-            "quickstart/samples-mux",
```

Comment:
let's keep samples-mux above samples-mysql as both use `Mux` difference being only the database used with them.